requirements-dev.txt

```diff
@@ -5,5 +5,5 @@ freezegun ; python_version != '3.3'
 hypothesis >= 3.30
 coverage
 mock ; python_version < '3.0'
-pep517 >= 0.5.0
+build >= 0.3.0 ; python_version >= '3.6'
```

Comment:
I don't think you need the version qualifier here

Reply:
This runs on Python 3.3, which is not supported by python-build I believe.

Reply:
Oh 😊 technically virtualenv also does not support that xD